OpenRA.Mods.Common/Traits/Player/ProductionQueue.cs

```diff
@@ -80,8 +80,8 @@ public class ProductionQueueInfo : ITraitInfo
 
 		public void RulesetLoaded(Ruleset rules, ActorInfo ai)
 		{
-			if (LowPowerSlowdown <= 0)
-				throw new YamlException("Production queue must have LowPowerSlowdown of at least 1.");
+			if (LowPowerModifier < 100)
```

Comment:
The previous check was needed because 0 or negative values would break the calculation.  IMO there is no reason not to disallow modifiers between 1 and 100 (implying a speed up on low power) here.